deal.II-toolchain/packages/adolc.package

```diff
@@ -0,0 +1,51 @@
+# use development version?
+#CANDI_ADOLC_FROM_GIT=1
+
+if [ -n "$CANDI_ADOLC_FROM_GIT" ]; then
+    # download git repository
+    VERSION=master
+    EXTRACTSTO=ADOL-C-master
+
+    NAME=adol-c.git
+    PACKING=git
+    SOURCE=https://gitlab.com/adol-c/
+
+else
+    # download release tarball
+
+    VERSION=2.6.3
+    NAME=ADOL-C-${VERSION}
+    EXTRACTSTO=ADOL-C-${VERSION}
+    SOURCE=https://www.coin-or.org/download/source/ADOL-C/
+    CHECKSUM=f78f67f70d5874830a1ad1c0f54e54f7
+    PACKING=.tgz
```

Comment:
Maybe you should set everything belonging to v2.6.3 as comment (for further use / updates of the origin) and give a short comment why we need your 2.6.4-rc1 (not official) version here.